core/src/main/scala/org/apache/spark/deploy/master/Master.scala

```diff
@@ -53,8 +53,6 @@ private[deploy] class Master(
   private val forwardMessageThread =
     ThreadUtils.newDaemonSingleThreadScheduledExecutor("master-forward-message-thread")
 
-  private val hadoopConf = SparkHadoopUtil.get.newConfiguration(conf)
```

Comment:
SPARK-4011 add `hadoopConf` and used by `rebuildSparkUI`,  SPARK-12299 Remove history serving functionality from Master and `hadoopConf` become unused